### DISCUSSION TITLE
After the last update `==3.14.0` -> `==3.15.0`, `distinct` remove from `rest_framework.compat`

### DISCUSSION THREAD

**Discussion by Djailla** [encode/django-rest-framework#9289] [Potential Issue]
Hi,

Since the last update, `from rest_framework.compat import distinct` no longer works.

Thanks and cheers,

Bastien

**Comment by TGoddessana** [comment]
What error occurs? Could you share the error message?

**Comment by Djailla** [comment]
```ImportError: cannot import name 'distinct' from 'rest_framework.compat' (/Users/b.vallet/Workspace/rackguru/api/.venv/lib/python3.11/site-packages/rest_framework/compat.py)```

**Reply by TGoddessana** [reply]
The error message is from this pr,

https://github.com/encode/django-rest-framework/commit/b99df0cf780adc3d65362a4425f9bb6d85410bcb

I think you can use queryset.distinct() directly
